pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -294,6 +306,104 @@ def simulate_and_vjp(
 # The set of supported observables.
 
 
+def stopping_condition(op: Operator) -> bool:
+    """A function that determines whether or not an operation is supported by ``lightning.kokkos``."""
+    # These thresholds are adapted from `lightning_base.py`
+    # To avoid building matrices beyond the given thresholds.
+    # This should reduce runtime overheads for larger systems.
+    if isinstance(op, qml.QFT):
+        return len(op.wires) < 10
```

Comment:
We can cover this with the `test_templates` module in time.

Reply:
Ok the next PR, will be cover